cmake/Modules/opm-core-prereqs.cmake

```diff
@@ -37,6 +37,7 @@ set (opm-core_DEPS
 	# DUNE dependency
 	"dune-common"
 	"dune-istl"
+	"opm-common REQUIRED"
```

Comment:
> we cannot do REQUIRED on the opm-common dependency right now as opm-common does not produce a library and all dependencies are assumed to contain one.

that's strange: opm-material also does not produce a library and ewoms requires via REQUIRED. Maybe a test needs to be defined instead?